public/app/features/dashboard/state/PanelQueryRunner.ts

```diff
@@ -58,16 +51,15 @@ export class PanelQueryRunner {
 
   /**
    * Returns an observable that subscribes to the shared multi-cast subject (that reply last result).
-   * Here the caller can also control if transformations should be applied or not
    */
-  getData(format: PanelDataFormat = PanelDataFormat.Frames, applyTransforms = true): Observable<PanelData> {
-    const ensureFormats = postProcessPanelData(format);
+  getData(): Observable<PanelData> {
+    const ensureFormats = postProcessPanelData();
```

Comment:
Now that `postProcessPanelData` is really the same for everything, do we need a special function?  Could this be *pre* processed before it is sent to the subject?

Reply:
yea, if it's something that always should be done I think that makes sense.

Reply:
thanks